config/default.yml

```diff
@@ -580,6 +580,18 @@ Layout/EmptyLineAfterMultilineCondition:
   Reference:
     - https://github.com/airbnb/ruby#multiline-if-newline
 
+Layout/EmptyLineAfterRequires:
+  Description: 'Add an empty line after requires.'
+  Enabled: pending
+  VersionAdded: '<<next>>'
+  EnforcedStyle: beginning_of_file_only
+  SupportedStyles:
```

Comment:
Are those names in line with other similar cops? 

I haven't checked them, but it seems to me that something like `always` and `beginning_of_file` might be better options. I'm also wondering if the beginning of file style makes sense in general, as I've rarely seen this in the wild.

Reply:
> something like `always` and `beginning_of_file` might be better options

Those names sounds better. I will change them that way.

Reply:
> I'm also wondering if the beginning of file style makes sense in general, as I've rarely seen this in the wild.

I too think this is a bit controversial.

For example, this would make the following code valid:

```ruby
require 'a'
require 'b'

require 'c'
require 'd'
class Foo
  # ...
end
```

Ideally, I would like the above code to look like this:

```ruby
require 'a'
require 'b'

require 'c'
require 'd'

class Foo
  # ...
end
```

So it might be better to nuance it as `top_level` instead of `beginning_of_file` and change the implementation accordingly 🤔

Reply:
Yeah, that seems better to me.